app/controllers/obituaries/community_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Obituaries::CommunityController < ApplicationController
   def index
-    @obituaries = Obituary.all
+    @obituaries = Obituary.all.paginate(page: params[:page], per_page: 10)
```

Comment:
I only set up pagination on the browsing indexes because I wasn't able to get it working on the tables.

Reply:
That's cool, I think for now this is great. Maybe can make an issue with some details on the project board for future us to know where to pick this back up if we want?

Reply:
that works, great job